tests/host/common/MockEsp.cpp

```diff
@@ -39,6 +39,8 @@
 #include <user_interface.h>
 struct rst_info resetInfo;
 
+char wifi_station_hostname [33]; // exists in nonosdk
+
```

Comment:
common.cpp still has `char wifi_station_get_hostname_str[128]`
mock should just use this array instead?